certbot-dns-yandex/certbot_dns_yandex/__init__.py

```diff
@@ -0,0 +1,85 @@
+"""
+The `~certbot_dns_yandex.dns_yandex` plugin automates the process of
+completing a ``dns-01`` challenge (`~acme.challenges.DNS01`) by creating, and
+subsequently removing, TXT records using the Yandex API.
+
+
+Named Arguments
+---------------
+
+========================================  =====================================
+``--dns-yandex-credentials``              Yandex credentials_ INI file.
+                                          (Required)
+``--dns-yandex-propagation-seconds``      The number of seconds to wait for DNS
+                                          to propagate before asking the ACME
+                                          server to verify the DNS record.
+                                          (Default: 30)
+========================================  =====================================
+
+
+Credentials
+-----------
+
+Use of this plugin requires a configuration file containing Yandex API
+credentials, obtained from your Yandex
+`account page <https://tech.yandex.com/domain/doc/concepts/access-docpage/#access-admin>`_.
```

Comment:
This page also talks about several different types of authentication. Because Yandex supports these, I think it'd be helpful to clearly document which method this plugin uses.

Usually we link the the specific page of the DNS provider's dashboard where the user can generate a token/key, but Yandex seems to have a more complicated process than most.

Perhaps we need to adjust the wording here to say something more like:
> Use of this plugin requires a configuration file containing Yandex API credentials. Follow the instructions in the [Yandex API documentation](https://tech.yandex.com/domain/doc/concepts/access-docpage/#access-admin) to obtain a PPD token.